data-science-onramp/ai-platform/test_models.py

```diff
@@ -0,0 +1,48 @@
+from collections import namedtuple
```

Comment:
this needs to be renamed `models_test.py`

Reply:
Actually, this should likely be split up into two files, one in each model directory with the related tests